src/main/java/org/osc/controller/nsc/api/SampleSdnRedirectionApi.java

```diff
@@ -87,11 +97,8 @@ public String installInspectionHook(NetworkElement inspectedPort, InspectionPort
         if (inspectedPort == null) {
             throw new IllegalArgumentException("Attempt to install an Inspection Hook with no Inspected Port");
         }
-        if (inspectionPort == null) {
```

Comment:
why are we removing all these checks?

Reply:
@arvindn, oh it is by accident and will fix it